packages/devextreme/js/__internal/scheduler/header/m_date_navigator.ts

```diff
@@ -40,7 +40,10 @@ export const getDateNavigator = (header, item) => {
 const getPreviousButtonOptions = (header) => ({
   key: 'previous',
   icon: 'chevronprev',
-  elementAttr: { class: PREVIOUS_BUTTON_CLASS },
+  elementAttr: {
+    class: PREVIOUS_BUTTON_CLASS,
+    ...getCurrentViewText(header) ? { 'aria-label': `Previous ${getCurrentViewText(header)}` } : {},
```

Comment:
This value should be calculated before. It is more friendly way to debug. 
https://refactoring.guru/extract-variable

Reply:
And the constants should be localizable, for example dxScheduler-NextMonth.
See the examples here: https://github.com/DevExpress/DevExtreme/blob/24_2/packages/devextreme/js/localization/messages/en.json#L252

Reply:
Do not forget to check them Vlada!